acceptance_tests/utilities/jwe_helper.py

```diff
@@ -16,3 +17,17 @@ def decrypt_signed_jwe(signed_jwe: str) -> Mapping:
 
     # Extract, deserialize, and return the payload
     return json.loads(jws_token.payload)
+
+
+def decrypting_token_and_asserts(context, query_strings: dict):
```

Comment:
Minor gripes, but I think functions like this with test specific code should live in the steps modules, rather than the helpers which are supposed to be for generic, non step specific interactions.